layouts/_default/baseof.html

```diff
@@ -5,6 +5,9 @@
         {{- partial "scriptsBodyStart.html" . -}}
         {{- partial "header.html" . -}}
         {{- block "main" . }}{{- end }}
+        {{ if .Site.Params.show_tag }}
```

Comment:
Why show tags on every page?
Particularly, I think it doesn't look good on single page.

![image](https://github.com/hugo-sid/hugo-blog-awesome/assets/122173059/9a28c862-ff68-4795-bf3e-2b9ba9195450)

I have another idea:
I guess your goal is to let a visitor, easily access tags. Let's have a link to /tags/ in the menu.